tests/apollo/util/bft.py

```diff
@@ -56,17 +57,25 @@ class ConsensusPathPrevalentResult(Enum):
    TOO_MANY_REQUESTS_ON_UNEXPECTED_PATH = 2
 
 KEY_FILE_PREFIX = "replica_keys_"
+# To run TLS/UDP clients, replace the next constant with a client implemented in bft_client.py 
+# For now the choices are UdpClient or TcpTlsClient:
+BFT_CLIENT_TYPE = bft_client.UdpClient
```

Comment:
Why is the UdpClient hard-coded here? How does it change if we want to run with TLS?

Reply:
This was a mistake I've already fixed an hour ago.
You are right.
It now looks at the environment to decide.